README.md

```diff
@@ -16,10 +16,16 @@ unzip BitBar-v1.9.1.zip && mv BitBar.app ~/Applications && open BitBar.app
 - Click on the BitBar icon
 - Set your favourite directory as Plugin Folder
 - Clone this repos into your plugin folder
-- Copy the script outside
+- Copy the script and the package.json on your Plugin Folder
```

Comment:
Right now the repo has more files: package.json, README.md... These will be interpret as plugin candidates by Bitbar